pkg/sql/sqlbase/testutils.go

```diff
@@ -1449,6 +1525,9 @@ func randColumnTableDef(rand *rand.Rand, tableIdx int, colIdx int) *tree.ColumnT
 	return columnDef
 }
 
+// randIndexTableDefFromCols creates an IndexTableDef with a random subset of
+// the given columns and a random direction. It has a 50% chance to create a
```

Comment:
I don't see where this function can create a partial index.

Reply:
Good catch. This is a stale comment from my first iteration which didn't use a mutator.

Reply:
Done.